### PR TITLE
add pr template

### PR DESCRIPTION
**Pull request by mouuii**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (xuzhu-591) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: How about using [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) ?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+```
+---
+name: "\U0001F680 Feature request"
```
Comment (kilosonc) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: What's the difference between `feature request` & `Enhancement Request`

File: .github/pr-title-checker-config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (kilosonc) [inline .github/pr-title-checker-config.json:1]: This won't take effect for these's no corresponding github actionshttps://github.com/marketplace/actions/pr-title-checker

Comment (mouuii) [reply .github/pr-title-checker-config.json:1]: deleted

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mouuii) [reply .github/ISSUE_TEMPLATE/feature_request.md:3]: deleted

**Review by kilosonc — APPROVED** [review]

**Review by xuzhu-591 — APPROVED** [review]

**Pull request merged**